### ISSUE TITLE
Not able to execute on Intellij Idea

### ISSUE THREAD

**Issue by occhioni-esteco** [Thizzer/jtouchbar#31]
Hi! 

I just want to use this library in order to check its usage but i am not able to run the code on intellij Idea due to a compile error:

```
JFrame frame = new JFrame();
JTouchBar jTouchBar = new JTouchBar();
jTouchBar.setCustomizationIdentifier("MySwingJavaTouchBar");
frame.setSize(400, 400);
frame.setDefaultCloseOperation(WindowConstants.EXIT_ON_CLOSE);
jTouchBar.show(frame);
frame.setVisible(true);
```

Calling the show() method will launch the compile error: 

> Error:(59, 18) java: cannot access org.eclipse.swt.widgets.Shell
>   class file for org.eclipse.swt.widgets.Shell not found

note. i have imported the jar with gradle. Fresh new project. 
    compile 'com.thizzer.jtouchbar:jtouchbar:0.1.0'

Tried to import also org.eclipse.swt library doesn't fix the problem. 

Thanks for your help

**Comment by occhioni-esteco** [comment]
Update: probably is due to a Gradle jar that is no more available on the repository which is 

org.eclipse.swt.cocoa.macosx.x86_64-4.3.jar

**Comment by ttveldhuis** [comment]
This should be solved in 0.2.0 with the separation of SWT support to a separate project. I hope to release that soon but for now the advice is to use the snapshot version of that build;

```
<dependency>
	<groupId>com.thizzer.jtouchbar</groupId>
	<artifactId>jtouchbar</artifactId>
	<version>0.2.0-SNAPSHOT</version>
</dependency>
```

**Comment by ttveldhuis** [comment]
Version 1.0.0 has been released and should resolve this issue. It might take a while until it is available in the repository.
